url_test.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/python3
```

Comment:
[https://floating-island-12885.herokuapp.com/](https://floating-island-12885.herokuapp.com/) This is the link. I wasn;t able to set the config vars. So i took random values. The url testing is working fine. Giving outputs on regular bases.